### PR TITLE
implicit tuple parameter unpacking is removed in python 3

### PR DESCRIPTION
**Pull request by ericvaandering**
See: https://www.python.org/dev/peps/pep-3113/

auto-fixed by futurize

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: FWCore/ParameterSet/python/TreeCrawler.py [hunk 1]
```diff
@@ -120,7 +120,8 @@ def import_module(self,partnam,fqname,parent):
         if r is not None:
             self._depgraph.setdefault(self._last_caller.__name__,{})[r.__name__] = 1
         return r
-    def load_module(self, fqname, fp, pathname, (suffix, mode, type)):
+    def load_module(self, fqname, fp, pathname, xxx_todo_changeme):
```
Comment (Dr15Jones) [inline FWCore/ParameterSet/python/TreeCrawler.py]: Could you provide a better name for the variable?

Comment (ericvaandering) [inline FWCore/ParameterSet/python/TreeCrawler.py]: That actually comes straight from the futurize conversion, but if you have a suggestion (I think this is code you own) I will be happy to update the branch.

Comment (Dr15Jones) [inline FWCore/ParameterSet/python/TreeCrawler.py]: aux_info?

File: RecoLuminosity/LumiDB/plotdata/create_public_lumi_plots.py [hunk 2]
```diff
@@ -417,10 +417,11 @@ def GetXLocator(ax):
 
 ######################################################################
 
-def TweakPlot(fig, ax, (time_begin, time_end),
+def TweakPlot(fig, ax, xxx_todo_changeme,
```
Comment (Dr15Jones) [inline RecoLuminosity/LumiDB/plotdata/create_public_lumi_plots.py]: time_range?

File: RecoLuminosity/LumiDB/plotdata/create_public_peakpu_plots.py [hunk 3]
```diff
@@ -126,10 +126,11 @@ def GetXLocator(ax):
 
 ######################################################################
 
-def TweakPlot(fig, ax, (time_begin, time_end),
+def TweakPlot(fig, ax, xxx_todo_changeme,
```
Comment (Dr15Jones) [inline RecoLuminosity/LumiDB/plotdata/create_public_peakpu_plots.py]: time_range?

**Comment by ericvaandering** [comment]
please test

(Will Jenkins listen to me?)

**Comment by ericvaandering** [comment]
I guess it does. @Dr15Jones I made your suggested changes. Thanks.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
@ericvaandering The tests completed for your original commit (the commit hash is at the top of the test complete message). I'm pretty sure if Jenkins was listening to your command it would have printed 'The tests are being triggered in jenkins'.

**Comment by Dr15Jones** [comment]
+1

**Comment by ericvaandering** [comment]
Want to start them or not? I can imagine you have a list of people allowed to start tests. 

Sent from a mobile device. Please excuse my brevity or transcription errors.

**Comment by Dr15Jones** [comment]
I did start them right before I made that comment :)

**Pull request merged**